core/src/callstate.js

```diff
@@ -1,37 +1,39 @@
-import daily from "./core.js";
-
 let state = {};
 let updateHandlers = [];
-let call = {};
 let waitingForSync = false;
+let call = null;
 const SYNC_MAX_WAITING_TIME = 7000;
 const UPDATE_MSG = "update-call-state";
 const REQUEST_MSG = "request-call-state";
 
 /* Public interface */
 export default {
-  onCallStateUpdate: function (key, callback) {
-    if (!updateHandlers[key]) {
-      updateHandlers[key] = [];
-    }
-    updateHandlers[key].push(callback);
-  },
+  afterCreateFrame,
```

Comment:
I think that's an errant comment suggesting it should be private on line 34. I'll remove it. That was left from the first version, where extensions essentially registered themselves by calling something like `daily.afterCreateFrame(...)` in a private function to set up an afterCreateFrame handler in the core extension; now, the actual extensions are registered in the `DailyIframe.createFrame` call.

But more importantly, I'm trying to move away from calling everything inside Core an 'extension', because it makes things confusing. Instead, I'm calling it the core 'module', and it does a few specific things explicitly, like [calling these other modules' `afterCreateFrame` functions](https://github.com/daily-solutions/extensions/blob/62514922faeea770999ed598ae8e9e6819d5cbd4/core/src/core.js#L42-L44). More on this below. :)